docs/vim-practice.md

```diff
@@ -71,7 +71,7 @@ Use ++g+g++ to jump to the beginning of the file, and ++shift+g++ (`G`) to
 jump to the last line of the file.
 
 Now try ++slash++, type in any word (or prefix of a word) and ++enter++.  This should move the cursor to the beginning of the word.  You can use
-++n++ and ++p++ to move to the next match and the previous match.
+++n++ and ++b++ to move to the next match and the previous match.
```

Comment:
> I think both `b` and `p` are not correct. Should be ++shift+n++.

I tried the exercise on my vim console and using `n` + `b` to navigate worked fine. Would you rather I change it to the following anyway?

```suggestion
++shift+n++ to move to the next match.
```